src/com/autoembed/autoembed.js

```diff
@@ -47,12 +53,20 @@ export default class AutoEmbed extends Component {
     }
 
     if (url.domain) {
+
+      /*
+      * Embeded content
+      */
       if (url.domain.indexOf('youtube.com') !== -1) {
         if ((url.paths && url.paths[0] == 'watch') && url.args.v) {
           return <YoutubeEmbed href={props.href} />;
```

Comment:
I realize this isn't from this PR (again, I'm taking notes for linter/style guide improvements), but prefer this:
```
if ( url.domain.indexOf('youtube.com') !== -1 ) {
          if ( url.paths && (url.paths[0] == 'watch') && url.args.v ) {
```
I mention spacing of special keywords elsewhere.

Of course use added brackets as needed, but a rule of thumb is to look at what's inside the conditional. 

* If it's a single conditional/implicit statement (i.e. `if ( url.paths && url.args.v ) {`) it's fine without added brackets
* If it's only a single `LHS == RHS` statement, it's fine without added brackets
* If multiple statements are used, separated by `&&` or `||`, and one is a `LHS == RHS` statement, use added brackets just to visually isolate it. i.e. `if ( other && (LHS == RHS) ) {`
* If the LHS or RHS side is more than a single item, use brackets to visually isolate the complexity. i.e. `if ( (a+b) == 12 ) {`

`&&` operations in series will always require the previous one to succeed, so no extra brackets are needed. When you mix and match `&&` with `||`, extra brackets will be necessary to guarantee order.

`&&`, `||`, assignment (`=`), and equality tests (`==`, `!==`, `>=`, etc) are special, and should be given extra spaces around them to make them easier to notice. 

Whether to space arithmetic operators (`+`, `-`, etc) is up to the person writing the math. If it's easier to follow with spaces, then use spaces.